### PR TITLE
Added Text In Json File. 

### PR DESCRIPTION
**Pull request by MeSophie**
Fixed Partially #7650
Fixed Partially #7649
Fixed Partially #7647
Fixed Partially #7602

@lloyddewit This PR replace #7875 please have a look.

### CODE REVIEW

File: instat/translations/en/r_instat_menus.json [hunk 1]
```diff
@@ -456,4 +456,4 @@
   "Rename": "Rename",
   "Save": "Save",
   "Import SST...": "Import SST...",
-  "Daily Data Editind/Entry...": "Daily Data Editind/Entry..."}
\ No newline at end of file
+  "Daily Data Editing/Entry...": "Daily Data Editing/Entry..."}
```
Comment (lloyddewit) [inline instat/translations/en/r_instat_menus.json:459]: This change is already in the master JSON file

File: instat/translations/en/r_instat_not_menus.json [hunk 2]
```diff
@@ -3659,5 +3659,34 @@
...
+  "Names To:": "Names To:",
+  "Values To:": "Values To:",
+  "Drop Prefix:": "Drop Prefix:",
+  "Drop Missing Values": "Drop Missing Values",
+  "Carry All Columns": "Carry All Columns",
+  "Columns to Carry": "Columns to Carry",
+  "Pivot Longer": "Pivot Longer",
+  "Unnest": "Unnest",
+  "names": "names",
+  "value": "value",
```
Comment (lloyddewit) [inline instat/translations/en/r_instat_not_menus.json:3672]: these changes are already in the master JSON file

**Review by lloyddewit — APPROVED** [review]
@MeSophie Thank you for these changes.
These files contain changes that are already in the master branch. I suspect that this PR branch comes from an older master commit (before the PR #7861 was merged). It's best to always pull from upstream master before making a new branch.
For this PR, it doesn't matter too much and I was able to upload the 19 real 'non-menu' changes to CrowdIn without any problem.
Thanks

**Comment by lloyddewit** [comment]
@N-thony @MeSophie 19 new 'non-menu' changes uploaded to CrowdIn, please could you translate to French?
(sorry, I forgot to make a screen shot)
Thanks

File: instat/translations/en/r_instat_not_menus.json (hunk 2 above)
Comment (N-thony) [reply instat/translations/en/r_instat_not_menus.json:3672]: yes

**Pull request closed without merging**